src/Sparrow/Json/JsonContextPool.cs

```diff
@@ -2,6 +2,10 @@
 {
     public class JsonContextPool : JsonContextPoolBase<JsonOperationContext>
     {
+        public JsonContextPool(Size? maxContextSizeToKeep = null) : base(maxContextSizeToKeep)
```

Comment:
create two separate ctors to not break binary compatibility

Reply:
done